### PR TITLE
fix ass_set_storage_size not be called (#7)

### PR DESCRIPTION
**Pull request by seiya-git**
fix ass_set_storage_size not be called #7

### CODE REVIEW

**Comment by seiya-git** [comment]
https://github.com/libass/libass/issues/591

**Comment by seiya-git** [comment]
fix #8

**Comment by TheOneric** [comment]
It seems the second commit updates libass to 0.15.2 and FreeType to 2.12.0; this will lead to crashes on Microsoft Windows. Until libass 0.16.0 is (soon) released fixing the incompatibility, better keep FreeType at a version <= 2.11.0.

File: README.md [hunk 1]
```diff
@@ -80,8 +80,8 @@ When no hint found in ASS script and 'colorspace' parameter is empty then the de
 
 ### Windows Visual Studio 2019
 
-* Prequisite: vsyasm integration
-  - get VSYASM from https://github.com/ShiftMediaProject/VSYASM
+* Prequisite: vsnasm integration
+  - get VSYASM from https://github.com/ShiftMediaProject/VSNASM
```
Comment (TheOneric) [inline README.md:84]: forgot to change VSYASM to VSNAMS at the start of the second line.

**Comment by seiya-git** [comment]
@TheOneric, thanks
I reverted FreeType to 2.11.0

**Comment by TheOneric** [comment]
> So after reverting freetype to 2.11[.0], this is safe to use and does not crash on Microsoft Windows?

There are no more known incompatibilities in relation to libass, yes. I can't guarantee this for other libraries, but it should be unlikely. On a related note libass 0.16.0 has just been released and once ShiftMediaProject has picked it up freetype and libass can both be updated to their latest release.
*(The second commit still bundles many unrelated changes though, which I personally would not want)*

In any case, the [first commit](https://github.com/pinterf/assrender/commit/93f8f860484b2f75ddd56b1db7e72c73b9090cd6) `fix ass_set_storage_size not be called` is definitely good to merge.

File: .github/workflows/github-ci.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      - name: install submodules and nasm
+        run: |
+          git submodule update --init --recursive --remote
+          git clone https://github.com/ShiftMediaProject/VSNASM.git
+          .\VSNASM\install_script.bat
+      - name: revert freetype to VER-2-11-0 (libass windows bug)
+        run: |
+          cd SMP/freetype2
+          git checkout VER-2-11-0
+          cd ../..
```
Comment (TheOneric) [inline .github/workflows/github-ci.yml:50]: Why did you just add this? The submodule should already point at SMP’s (second) version of freetype’s 2.11.0 release according to your previous comment and the changelog iinm.

**Comment by seiya-git** [comment]
@TheOneric, I'd better make them to separate pull requests

**Comment by TheOneric** [comment]
@pinterf:  With only the first commit left, this is now definitely safe to apply :)

@seiya-git: You might want to wait a few days or so until ShiftMediaProject has picked up libass 0.16.0 before resubmitting your other changes as a new PR. I'd also suggest to split these chacnges into three separate commits (SMP update, CI and release bump) but ultimately that's for pinterf to decide.

**Pull request merged**